enos/enos.py

```diff
@@ -330,6 +377,11 @@ def cartesian(d):
             product.append(dict(e))
         return product
 
+    # Override default result directory name when --env parameter is given
```

Comment:
This piece of code seems duplicated 4 times. Is it possible to factorize it somehow ?